src/Bridge/Symfony/Bundle/Resources/config/graphql.xml

```diff
@@ -15,6 +15,7 @@
             <argument type="service" id="serializer" />
             <argument type="service" id="api_platform.metadata.resource.metadata_factory" />
             <argument type="service" id="api_platform.security.resource_access_checker" on-invalid="ignore" />
+            <argument type="service" id="api_platform.name_converter" on-invalid="ignore" />
```

Comment:
I think we could add it before and remove the `on-invalid`. There is no BC policy for GraphQL and we have already done a lot of BC breaks in master anyway.

Reply:
Actually there's not always a name_converter in every app, that's why it is nullable @alanpoulain, not really for BC reason bit I might be wrong...

Reply:
OK. Could you just change its position then (before a parameter for instance)?

Reply:
Well optional parameters are always supposed to be the last ones, and parameter are not optional unless I'm wrong @alanpoulain ?

Reply:
Because you are adding `on-invalid="ignore"`, it will be replaced by `null` if the service doesn't exist.
IMO we have to use an optional parameter at the last and with a default value only for BC policy: we don't want a decorator to break for instance.